src/routes/ChatFinished/container.js

```diff
@@ -11,9 +12,12 @@ export class ChatFinishedContainer extends Component {
 		route('/');
 	}
 
-	render = (props) => (
-		<ChatFinished {...props} onRedirectChat={this.handleRedirect} />
-	)
+	render = (props) => {
+		if (!isMobile()) {
+			return route('/');
+		}
+		return <ChatFinished {...props} onRedirectChat={this.handleRedirect} />;
+	}
```

Comment:
IMO We should handle this in `onChatClose` function instead of here. `ChatFinished` is a separate component. It should be `onChatClose` function's responsibility whether we want to route to `/chat-finished` or `/`

https://github.com/WideChat/Rocket.Chat.Livechat/blob/develop/src/lib/room.js#L41...L45